Test handlePacket_keepalive fails

**Describe the bug**
```
--- FAIL: TestRouter_handleTransportPacket (2.23s)
    --- FAIL: TestRouter_handleTransportPacket/handlePacket_keepalive (0.12s)
        router_test.go:366: 
            	Error Trace:	router_test.go:366
            	            				router_test.go:344
            	Error:      	Received unexpected error:
            	            	routing table: rule keep-alive timeout exceeded
            	Test:       	TestRouter_handleTransportPacket/handlePacket_keepalive
```

Test fails in this build https://travis-ci.com/github/skycoin/skywire/jobs/401873214#L2844

--- a/pkg/router/router_test.go
+++ b/pkg/router/router_test.go
@@ -367,16 +367,16 @@ func testKeepAlivePacket(t *testing.T, r0, r1 *router, pk1, pk2 cipher.PubKey) {
 	require.NoError(t, err)
 	require.Len(t, r0.rt.AllRules(), 1)
 
-	time.Sleep(50 * time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
 
 	packet := routing.MakeKeepAlivePacket(rtIDs[0])
 	require.NoError(t, r0.handleTransportPacket(context.TODO(), packet))
 
 	require.Len(t, r0.rt.AllRules(), 1)
-	time.Sleep(40 * time.Millisecond)
+	time.Sleep(10 * time.Millisecond)
 	require.Len(t, r0.rt.AllRules(), 1)
 
-	time.Sleep(110 * time.Millisecond)
+	time.Sleep(200 * time.Millisecond)
 	require.Len(t, r0.rt.AllRules(), 0)
 }
 

